tasks/create-internal-request/create-internal-request.yaml

```diff
@@ -53,6 +53,40 @@ spec:
     - name: input
       description: Workspace to store the params and responses for the internalRequest
   steps:
+    - name: merge-internal-request-parameters
+      image:
+        quay.io/hacbs-release/release-utils@sha256:5733ece907aa70d6ebced36484c936731e8b27bfcf87fed226a0ecf95047a6b8
+      script: |
+          #!/usr/bin/env bash
+          DATA_FILE="$(workspaces.input.path)/$(params.dataPath)"
```

Comment:
does this have the `pipeline.context.id` in it?

Reply:
I personally have been putting that only in the pipelines as I think it makes more sense there (and doesn't make much sense in the task if you think about running the task standalone). For example, https://github.com/redhat-appstudio/release-service-catalog/blob/main/pipelines/push-to-external-registry/push-to-external-registry.yaml#L234 but the `create-pyxis-image` uses identical code to this

Reply:
Yes, it has it https://github.com/redhat-appstudio/release-service-catalog/pull/240/files#diff-3778095b07e9be9d621f33c7b73cd675da51308dd923e44c58458738d8c023cdR306

Reply:
I see it now.